js/scripts.js

```diff
@@ -79,4 +78,385 @@ document.querySelector('.contact_mobile').addEventListener('click',() =>{
+
+            <h2 class="grid-title">${projects[i].title}</h2>
+            <ul class="experience-container">
+              ${li1}
+            </ul>
+            <p class ="project-description"> 
+              ${projects[i].description}
+            </p>
+            <div class = "technologies"
+              <ul class = "tech-container>
+                ${li2}
+              </ul>
+            </div>
+          
+              <div class="action-grid">
+                <button whichProjectBtn = ${i} class="action-grid-btn" type = "button">
+                  See Project
+                </button>
+              </div>
+          
+          </div>`;
+    gridsContainer.append(gridDiv);
+  }
+
+  // Popup window for mobile version starts here
+  const popupWindow = document.querySelector('.popup-card-wrapper');
+  function showPopupWindow() {
+    const btn = document.querySelectorAll('.action-grid-btn');
+    btn.forEach(function (e) {
+      e.addEventListener('click', function() {
+        const index = e.getAttribute('whichProjectBtn');
+        popupWindow.classList.toggle('hide');
+        const experiencePopup = projects[index].experiences;
+        const techpopup = projects[index].technologies;
+        let li1 = '';
+        for (let i = 0; i < experiencePopup.length; i++) {
+          li1 += `
+            <li class="popup-exp-name">${experiencePopup[i]}</li>
+          `;
+        }
+        let li2 = '';
+        for (let j = 0; j < techpopup.length; j++) {
+          li2 += `
+            <li class="popup-exp-name">${techpopup[j]}</li>
+            `;
+        }
+        popupWindow.innerHTML = `
+        <div class=".popup-card">
+          <div class="popup-title-and-closeIcon-container">
+            <h2 class="popup-project-title"> ${projects[index].title}
+            <h2>
+            <i class="fa fa-times closePopup"></i> 
+          </div>
+          <ul class="popup-experience">
+            ${li1}
+          </ul>
+          <img class = "main-image" src="${projects[index].featuredImage}" alt="Tonic Image">
+          <p class = "popup-card-text">${projects[index].description}</p>
+          <ul class="popup-technologies">
+            ${li2}
+          </ul>
+          <div class="popup-button-group">
+            <div class="popup-knowBtns">
+              <button type="button" class="popup-btn-text">See live</button>
+              <img src="./images/popup/Icon-Export-btn1.png" alt="">
+            </div>
+            <div class="popup-knowBtns">
+              <button type="button" class="popup-btn-text">See source</button>
+              <img src="./images/popup/Icon-GitHub-popbt2.png " width="24px" height="24px" alt="">
+            </div>
+          </div>
+        </div> `;
+
+        
+        
+          /* const popupCardWrapper = document.querySelector('.popup-card-wrapper');
+             const popupCard = document.querySelector('.popup-card');
+             popupWindow.append(popupCard);
+          */
```

Comment:
- [Optional] kindly remove all unused code from your documents. It is best practice not to leave commented/unused code in your documents.